config/config.go

```diff
@@ -0,0 +1,37 @@
+package config
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/tidepool-org/platform/Godeps/_workspace/src/github.com/jinzhu/configor"
+)
+
+//FromJSON will load the requested JSON config into the given config interface
+func FromJSON(config interface{}, name string) error {
```

Comment:
This code will likely change with my comments about per-environment configuration in each config file above.